### ISSUE TITLE
Load only Compiled JS for the specific page.

### ISSUE THREAD

**Issue by theArtechnology** [solidjs/solid-start#31]
It would be nice to have a feature where only the xx.data.js + its page xx.jsx are loaded for requested page instead of having the whole js of the `pages` in one file bundle.

Observation from: http://solidjs.github.io/solid-hackernews

Same for css

**Comment by ryansolid** [comment]
I think you are mistaken or I'm misunderstanding your request. The way it works are the data functions become part of the main bundle(`index.js`) but the pages end up in their own that are lazy loaded (ie.. `[stories].js`). The reason for this is to allow parallel data fetching when navigating the routes. We can fetch data and page code at the same time even in nested routes instead of causing client-side waterfalls. 

CSS should split already as well. The Hackernews demo just has global CSS so I did it that way when I copied it, but if you import it in the page it won't be loaded until that point. Alongside current route preload tags on SSR, to my knowledge we already have the optimal setup here.
